tests/test_api.py

```diff
@@ -497,6 +497,30 @@ def test_comment_staff(self):
         self.assert_403(response)
         assert response.json['code'] == 403
 
+        def test_get_comments(self):
+            self._test_backup(True)
+            user = User.lookup(self.user1.email)
+            backup = Backup.query.filter(Backup.submitter_id == user.id).first()
+            comment_url = "/api/v3/backups/{}/comment/".format(encode_id(backup.id))
+
+            #check to see if student can view comments on own backup's comments
+            self.login(self.user1.email)
+            response = self.client.get(comment_url)
+            self.assert_200(response)
+            self.logout()
+
+            #check to see if staff can access comments
+            self.login(self.staff1.email)
+            response = self.client.get(comment_url)
+            self.assert_200(response)
```

Comment:
You might also want to verify that the comment text matches what you would expect. That might require you to create a comment for that backup beforehand. Feel free to look at how the function above does it - but a better way might be to create an instance of the Comment model - fill in the appropriate data (like what backups it belongs to and what the contents are) 

```
comment = Comment(
        backup_id=...,
        ... all of the other fields in model.py:Comment )
 db.session.add(comment)
```